server/webdriver_tests/shared.py

```diff
@@ -47,3 +50,18 @@ def click_sv_group(driver, svg_name):
     if svg_name in group.text:
       group.click()
       break
+
+
+def wait_for_charts(driver):
```

Comment:
nit: this function isn't doing the waiting, so maybe "charts_rendered" or something

Reply:
Done